resources/views/components/card/address.blade.php

```diff
@@ -0,0 +1,37 @@
+@props(['shipping' => false, 'billing' => false, 'dynamicType' => false, 'check' => false])
+
+<address-card
+    {{ $attributes->whereStartsWith('v-') }}
+    @if(!$dynamicType)
+        :shipping="{{ var_export($shipping) }}"
+        :billing="{{ var_export($billing) }}"
+    @endif
+    v-slot="{ address, billing, shipping, isEmpty }"
+>
+    <div {{ $attributes->whereDoesntStartWith('v-')->class('relative bg-white border px-8 py-6 rounded max-w-[400px]') }} v-if="!isEmpty">
+        <div class="h-full flex flex-col">
+            @if($check)
+                <template v-if="{{ $check }}">
+                    <div class="absolute w-1 left-0 inset-y-0 rounded-l bg-accent"></div>
```

Comment:
please check this whole file for colors and variables not being prefixed with ct-